core/vm/evm.go

```diff
@@ -669,13 +669,12 @@ func (evm *EVM) PublishEvent(
 ) {
 	context := evm.Context
 	if context.CurrentTransaction == nil {
-		log.Debug("[EVM] PublishEvent - Transaction is nil", "height", context.BlockNumber.Int64())
```

Comment:
I think we still could keep this debug log for tracking logic currentTransaction is nil.